Math/countPrimes.java

```diff
@@ -0,0 +1,56 @@
+/* Implement countPrimes(n), which calculates the number of primes that are strictly smaller than n.
```

Comment:
The code looks good, could you also update Time and Space complexity? 
Keep going and pick more issues 
Thanks

Reply:
Yes I'm on it.